src/main/resources/application.properties

```diff
@@ -15,3 +15,6 @@ stairway.migrateUpgrade=true
 stairway.forceCleanStart=true
 stairway.quietDownTimeout=20s
 stairway.terminateTimeout=5s
+pubsub.track-resource.pubsubEnabled=${TRACK_RESOURCE_PUBSUB_ENABLED}
+pubsub.track-resource.projectId=${TRACK_RESOURCE_PUBSUB_TOPIC}
```

Comment:
the property is the project id, but the env variable is for the topic. It doesn't seem like the service uses the topic in the subscription at all. what's up with that?
the fact that this isn't caught be any test makes me think that we should change our test configuration to set these environment variables instead of overriding them in the application-integration-test.properties.

Reply:
ah, sorry, this should be the project id. It get messed up when I tried to add topic & project & subscription then deleted the topic. Didn't find it out because I haven't deployed this to k8s yet.  It works in test because in test we stetted it correctly there.  We don't need topic because subscription is created in Terraform.
Sorry, I didn't capture that because I havn't deploied that to my env yet, I can set those environment variable in gradle. Do you want me set all of them there, including db uri, password, username, stairway stuff or just pubsub stuff?

Reply:
Eh, I feel like we should have a test at some point that captures configuration errors like this. That could be the 'deploy to a ephemeral k8s environment integration test,' but earlier is often better. Maybe we should include all the environment variables set in our deployment.yaml as a part of gradle for the integration tests. That feels like it's a little closer to what's going on, but on the other hand, that's only for the integration tests, not the unit or local runs. WDYT?